scala/dllib/src/main/scala/com/intel/analytics/bigdl/dllib/example/nnframes/lightGBM/LgbmClassifierTrain.scala

```diff
@@ -40,7 +40,7 @@ object LgbmClassifierTrain {
         StructField("petal length", DoubleType, true),
         StructField("petal width", DoubleType, true),
         StructField("class", StringType, true)))
-      val df = spark.read.schema(schema).csv(params.inputPath)
+      val df = spark.read.schema(schema).csv(params.inputPath).repartition(sc.defaultParallelism)
```

Comment:
Since not every user wants to enable parallel training, pls set this to a configurable option
And as for the partition number, maybe the count of executors is better than default parallelism